### DISCUSSION TITLE
Avoid upstream unpartitioned asset invalidating all downstream partitions

### DISCUSSION THREAD

**Discussion by the4thamigo-uk** [dagster-io/dagster#19826] [Q&A]
Hi, I wonder how I can avoid an unpartitioned upstream asset making all partitions of a partitioned downstream asset stale?

**Comment by sryza** [comment]
Hey @the4thamigo-uk - this currently isn't possible, though it's a reasonable request. We've been talking about possibly making the "Upstream Data" label more generally customizable. Would you be up for filing a Github issue with this request?

**Reply by ion-elgreco** [reply]
@sryza somewhat related issue that I created, but highlights how disparate the worlds of unpartitioned and partitioned are in dagster, and that you can't really use them together https://github.com/dagster-io/dagster/issues/19731
